src/main/content/_layouts/doc.html

```diff
@@ -25,68 +25,66 @@
  - limitations under the License.
  -
  ------------------------------------------------------------------------------>
+ {% assign t = site.data[site.active_lang].general-reference %}
```

Comment:
To keep things consistant with how our translation data is laid out we should change this line to `{% assign t = site.data[site.active_lang].doc %}` and in the en.yml file change "general-reference" to "doc"